styles/styles.css

```diff
@@ -10,46 +10,101 @@
  * governing permissions and limitations under the License.
  */
 
- :root {
-  /* colors */
-  --link-color: #035fe6;
-  --link-hover-color: #136ff6;
-  --background-color: #fff;
-  --overlay-background-color: #eee;
+@font-face {
+  font-family: AvenirNextLTPro-Regular;
+  src: url("/fonts/AvenirNextLTPro-Regular.woff2") format("woff2");
+  font-display: swap;
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
```

Comment:
@solaris007 could you find references for `OpenSans-Regular` somewhere?
It seems that this is carried over by some underlying Wordpress plugin, but I can't find it used anywhere in the default viewport on mobile, unlike `PTSans-Regular` for the nav and `AvenirNextLTPro-Regular` for the content.